Packs/BmcITSM/Integrations/BmcITSM/README.md

```diff
@@ -1270,10 +1271,235 @@ Gets the list of incidents that were modified since the last update time. Note t
+#### Context Example
+```json
+{
+    "BmcITSM": {
+        "WorkOrder": {
+            "CreateDate": "2024-02-07T08:08:23",
+            "DisplayID": "WO0000000001002",
+            "RequestID": "WO0000000000702"
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Work order ticket successfully created.
+>|Create Date|Display ID|Request ID|
+>|---|---|---|
+>| 2024-02-07T08:08:23 | WO0000000001002 | WO0000000000702 |
+
+
+### bmc-itsm-work-order-update
+
+***
+Updates the work order ticket.
+
+#### Base Command
+
+`bmc-itsm-work-order-update`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| request_id | The ID of the work order ticket to update. | Required | 
+| summary | The work order ticket summary. | Optional | 
+| detailed_description | The work order ticket detailed description. | Optional | 
+| priority | The work order ticket priority. Possible values are: Critical, High, Medium, Low. | Optional | 
+| status | The work order ticket status. Possible values are: Assigned, Pending, Waiting Approval, Planning, In Progress, Completed, Rejected, Cancelled, Closed. | Optional | 
+| status_reason | The reason for changing the ticket status. Possible values are: Initial Status, Awaiting Request Assignee, Client Hold, Client Additional Information Requested, Client Action Required, Support Contact Hold, Local Site Action Required, Purchase Order Approval, Supplier Delivery, Third Party Vendor Action Required, Infrastructure Change, Work not started, Successful, Successful with Issues, Cancelled by Requester, Cancelled by Support, Customer Close, System Close, System Close with Issues. | Optional | 
+| company | The company associated with the requester. By default it is determined by the logged in user. | Optional | 
+| location_company | The company associated with the work order process. | Optional | 
+| assignee | The full name of the employee the work order is assigned to. It can be retrieved by using the bmc-itsm-user-list command. | Optional | 
+| support_organization | The organization for the problem assignee's support organization. It makes up the second tier of the problem assignee's support organization data structure. The arguments support_organization, support_group should be provided together. It can be retrieved by using the bmc-itsm-support-group-list command. | Optional | 
+| support_group | The group for the problem assignee's support group.  It makes up the third tier of the problem assignee's support organization data structure. The arguments support_organization, support_group should be provided together. It can be retrieved by using the bmc-itsm-support-group-list command. | Optional | 
```

Comment:
```suggestion
| support_group | The group for the problem assignee's support group. It makes up the third tier of the problem assignee's support organization data structure. The arguments support_organization, support_group should be provided together. It can be retrieved by using the bmc-itsm-support-group-list command. | Optional | 
```